symfony/framework-bundle/3.3/config/packages/test/framework.yaml

```diff
@@ -2,3 +2,7 @@ framework:
     test: ~
     #session:
     #    storage_id: session.storage.mock_file
+
+services:
+    # Disable logs
```

Comment:
As soon as Monolog is used with a proper stream handler for instance, registering this null logger probably isn't a good idea.

Reply:
Why? It will speed up the builds, even with Monolog.

Reply:
Logs can be useful to understand why a test fail. I'm not sure about this as is in the recipe.

Reply:
Anyway monolog has no specific config for tests right now.
It's just a recipe, for specific needs, debugging etc, the user can (should?) modify this file to fit his needs.

Reply:
the question then is whether this logger is winning over monolog or no. This should be checked
The goal is to have it winning over the default logger added in FrameworkBundle when no logger is defined

Reply:
or maybe we should make it possible to disable the default logger via config